core/src/com/unciv/ui/pickerscreens/TechPickerScreen.kt

```diff
@@ -60,7 +61,7 @@ class TechPickerScreen(internal val civInfo: CivilizationInfo, centerOnTech: Tec
 
         createTechTable()
         setButtonsInfo()
-        topTable.add(techTable)
+        topTable.add(techTable).fill().expand()
```

Comment:
Again, why both .fill() and .expand()? Doesn't .fill() do both?

Reply:
if you only use fill(), you will find the techTable can't cover stage entirely. because the padding we calculate according to the button's height is not very accurate, the left blank we fill with **"era.toLabel()"**.
You can find in the line 117, I also use fill() and expand().